.github/CHANGELOG.md

```diff
@@ -25,6 +25,45 @@
   sf.plot_wigner(state, mode, xvec, pvec, renderer="browser")
   ```
 
+* Fock state marginal probabilities can be plotted directly via Strawberry
+  Fields using Plot.ly.
+  [(#510)](https://github.com/XanaduAI/strawberryfields/pull/510)
+
+  ```python
+  prog = sf.Program(1)
+  eng = sf.Engine('fock', backend_options={"cutoff_dim":5})
+
+  with prog.context as q:
+      Sgate(0.5) | q[0]
+
+  state = eng.run(prog).state
+  state.all_fock_probs()
+
+  modes = [0]
+
+  sf.plot_fock(state, modes, cutoff=5, renderer="browser")
```

Comment:
The latex when hovering over the bars doesn't seem to work properly for me. :thinking:
![fock](https://user-images.githubusercontent.com/6934626/102529500-f9b60280-406d-11eb-919d-166629ffcf56.png)

Reply:
Otherwise it looks great. :smile:

Reply:
Yeah, seems like an [open `plotly` issue](https://github.com/plotly/plotly.js/issues/559).

Reply:
Oh, I see. :disappointed: